config/config.go

```diff
@@ -1,35 +1,66 @@
 package config
 
 import (
-	"fmt"
 	"log"
-	"path/filepath"
-
-	"github.com/spf13/viper"
+	"os"
 )
 
-// Init is an exported method that takes the environment starts the viper
-// (external lib) and returns the configuration struct.
-func Init(cfgFile string) {
-	if cfgFile != "" { // enable ability to specify config file via flag
-		viper.SetConfigFile(cfgFile)
-	} else {
-		viper.SetConfigName(".taco") // name of config file (without extension)
-		viper.AddConfigPath("$HOME") // adding home directory as first search path
+type Config struct {
+	AWS_Region  string
```

Comment:
Should these be specific to DynamoDB? e.g.: AWS_DYNAMODB_REGION

Reply:
Sure. Do we expect at some point services are likely to be in separate regions? I can make this change to be ready in that case.

Reply:
no, the container and the resources that it connects to should all be in the same region

Reply:
@eefahy So, if we use kinesis and dynamo, they must all be in the same region?

Reply:
yes